src/Autocomplete/src/DependencyInjection/AutocompleteExtension.php

```diff
@@ -155,4 +155,19 @@ private function registerFormServices(ContainerBuilder $container): void
                 new Reference('ux.autocomplete.entity_search_util'),
             ]);
     }
+
+    private function isAssetMapperAvailable(ContainerBuilder $container): bool
+    {
+        if (!interface_exists(AssetMapperInterface::class)) {
+            return false;
+        }
+
+        // check that FrameworkBundle 6.3 or higher is installed
+        $bundlesMetadata = $container->getParameter('kernel.bundles_metadata');
+        if (!isset($bundlesMetadata['FrameworkBundle'])) {
+            return false;
+        }
+
+        return is_file($bundlesMetadata['FrameworkBundle']['path'].'/Resources/config/asset_mapper.php');
```

Comment:
@stof I don't love this solution, but I can't think of a better option. I'm wondering if, in asset-mapper 6.4, we should allow a service-based way of adding "paths" instead of relying on prepending.

Reply:
I don't have a better idea for that for now